### ISSUE TITLE
SSL Error querying qdrant vector db

### ISSUE THREAD

**Issue by peter-boucher** [langflow-ai/langflow#3166]
### Bug Description

When using the Qdrant module in a flow based on the template "Vector Store RAG" I am able to ingest my data into a qdrant instance running locally in docker. When running the RAG flow, querying the vector store fails with an SSL record layer failure message. Screenshot:
![image](https://github.com/user-attachments/assets/cee63899-0929-4639-9556-6551085edf4d)

Python error
`ResponseHandlingException: [SSL] record layer failure (_ssl.c:1000)
/Users/peterboucher/projects/langflow/.venv/lib/python3.12/site-packages/pydantic/main.py:1059: PydanticDeprecatedSince20: The `__fields__` attribute is deprecated, use `model_fields` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.8/migration/
  warnings.warn(`

Qdrant log:
`qdrant  | 2024-08-02T13:34:38.637910Z ERROR actix_http::h1::dispatcher: stream error: request parse error: invalid Header provided
`

### Reproduction

Prerequisite: docker container running qdrant vectorDB

dockerfile.yml
```yaml
services:
  qdrant:
    image: qdrant/qdrant:latest
    restart: always
    container_name: qdrant
    ports:
      - 6333:6333
      - 6334:6334
    expose:
      - 6333
      - 6334
      - 6335
    volumes:
      - ./qdrant_data:/qdrant_data%
```
 1. Reproduce the flows pictured above (based on the template "Vector Store RAG"
 2. Using any PDF file as input. Run the lower ingestion flow, by clicking the play button on the last module Qdrant
 3. observe a collection is created in qdrant named 'langflow_demo'
 4. Run the upper RAG query flow by opening the Playground and typing a query related to the PDF content
 5. observe the error "[SSL] record layer failure"

### Expected behavior

 Expected a response from the model based on the ingested document in the vector store.

### Who can help?

@carlosrcoelho 

### Operating System

MacOS Sonoma 14.5 (23F79)

### Langflow Version

1.0.14

### Python Version

3.12

### Screenshot

![image](https://github.com/user-attachments/assets/cee63899-0929-4639-9556-6551085edf4d)

### Flow File

[RAG Demo.json](https://github.com/user-attachments/files/16471291/RAG.Demo.json)

**Comment by carlosrcoelho** [comment]
@peter-boucher 
Ensure that the Qdrant server is running. You can start the Qdrant server with Docker using the following command:
`docker run -p 6333:6333 -p 6334:6334 qdrant/qdrant`

**Comment by carlosrcoelho** [comment]
Thank you for your contribution! This issue will be closed. If you have any questions or encounter another problem, please open a new issue and we will be ready to assist you
